e2e/cypress/integration/onboarding/use_team_invite_link_to_sign_up_spec.js

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @onboarding
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+import {getEmailUrl, getEmailMessageSeparator, reUrl, getRandomId} from '../../utils';
+
+describe('Onboarding', () => {
+    let testTeam;
+    const randomId = getRandomId();
+    const username = `user${randomId}`;
+    const email = `user${randomId}@sample.mattermost.com`;
+    const password = 'passwd';
```

Comment:
How about exporting the [generateRandomUser](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/api/user.js#L139-L150) and import it here? So you don't need to define it again.